tests/DbaaS/pages/dbaasActionsPage.js

```diff
@@ -5,13 +5,13 @@ const assert = require('assert');
 
 module.exports = {
 
-  async checkActionPossible(actionName, actionPosibilty) {
+  async checkActionPossible(actionName, actionPosibilty, dbclusterName) {
```

Comment:
Please add JS doc with description and explanation of the arguments(please include complete list of values for the first argument). Also as I see second argument is boolean please apply any naming convention to make it obvious the variable/argument is boolean. I assume this argument is `expectedResult` is it correct?